buildbuddy.yaml

```diff
@@ -10,3 +10,5 @@ actions:
           - "main"
     bazel_commands:
       - "test --config=workflows //tools/... //test/..."
+      # GH126: Enable the Buildifier check once the current lint issues are addressed.
+      # - "run //:buildifier.check"
```

Comment:
We can probably put this under a new action, and not limit it to `darwin`, to allow it to run on linux.

Reply:
I moved it to its own action. Does that look about right? Of course, we won't enable this until the lint issues are addressed.